README.md

```diff
@@ -386,3 +285,56 @@ python3 -m generative_data_prep pipeline --input_file_path=./tests/examples/meta
 ```
 
 > [View decoded output](tests/examples/metaICL/decoded_data_prepped_metaICL.txt)
+
+</br>
+
+## Understanding Command Outputs
+
+### Terminal Output
+The metrics associated with this dataset will be printed in the terminal. These metrics give some insight into how the data was packed into sequences, and information about the training dataset.
+| Metric Name      | Definition | How to Interpret? |
+| --------- | --------- | --------- |
+| Articles | The number of lines in the input dataset. | How many text documents in the input dataset. |
+| Dataset Tokens | Number of tokens in the output hdf5 dataset. | How many tokens are in the training dataset. But this includes both prompt tokens and padding tokens, so this metric does not necessarily show how many tokens will learned by the model. |
+| Prompt Tokens | Number of prompt tokens in the output hdf5 dataset. | <- |
+| Completion Tokens | Number of completion tokens in the output hdf5 dataset. | <- |
+| Padding Tokens | Number of padding tokens in the output hdf5 dataset. | <- |
+| Average Completion Length | Number of completion tokens divided by number of input articles. | How long the average completion is the dataset. |
```

Comment:
How long the average completion is the dataset.
-> 
How long the average completion is in the dataset.